app-frontend/packs/components/dashboard/homePage/HomePageBanner.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+
+    <div
+        class="tt-home-page-banner">
+
+        <h2
+            class="tt-text tt-home-page-banner__title">
+            Name banner
+        </h2>
+
+        <p
+            class="tt-text tt-home-page-banner__text">
+            Name banner
+        </p>
+
+        <p
+            class="tt-text tt-home-page-banner__target-product">
+            Name banner
+        </p>
+
+        <Button
+            class="tt-home-page-banner__button tt-home-page-banner__button--first"
+            :type="'button'">
```

Comment:
статические пропы можно передавать без динамики

```suggestion
            type="button"
        >
```

Reply:
також рекомендую сінгл-атрібут елементи писати в одну строчку, і тільки мульті-атрібутні розділять так як ти написав